packages/protocol/test2/LibFixedPointMath.t.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
```

Comment:
This belongs to a third party, so maybe keep it UNLICENSED

Reply:
Changed.